server/main.lua

```diff
@@ -16,278 +9,6 @@ end
 TriggerEvent('esx_phone:registerNumber', 'mechanic', _U('mechanic_customer'), true, true)
 TriggerEvent('esx_society:registerSociety', 'mechanic', 'mechanic', 'society_mechanic', 'society_mechanic', 'society_mechanic', {type = 'private'})
 
-local function Harvest(source)
```

Comment:
Dont remove, just comment out the lines for debugging purposes.

Reply:
Same po with the previous comment, server/main.lua lines were restored, disabled functions po sa client side as suggested